app/frontend/src/components/admin/inspection/LocationEdit.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <v-dialog v-model="dialog" max-width="800px">
```

Comment:
Don't we have the BaseDialog.vue component in silvipc? We should be leveraging that component if possible.

Reply:
it doesnt really work for a form